content/articles/candian-bitcoin-owners-statistics-2021.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Bitcoin ownership among Canadians in 2021 
+description: According to the Bank of Canada’s Financial System Review, cryptoassets market have seen rapid growth and integration into the traditional finance in recent years
+img: https://user-images.githubusercontent.com/59807305/195985948-607de861-6ce7-41f5-813e-a777403a11bd.jpg
+alt: Image by cryptostock from Pixabay 
+author: 
+  name: B. Antonia
+  bio: |
+    🧠 Neuroscience student
+    🦸🏼 Goal digger
+  img: https://user-images.githubusercontent.com/1257310/190924545-66cd79f4-445a-41d5-9cd4-f29d00d3619c.jpg
+tags:
+  - canada
+  - bitcoin
+  - finance
+date: 2022-09-18T09:00:00.000Z
+draft: false
+---
+
+
+> The Bank of Canada has been studying Bitcoin since 2016 using surveys. The 2021 Bitcoin Omnibus Survey provides primarily information about awareness, ownership and usage of Bitcoins among Candadians. 
+
+## Five Takeawyas from the 2021 Bitcoin Omnibus Survey
+
+### Sharp increase of Bitcoin ownership in 2021
+Ownership of Bitcoin in 2021 has seen a steep rise in 2021 after keeping under 5% from 2018 to 2020. The report also uncovered that "Purchasers used mobile apps much more than web-based exchanges, mining or visiting a Bitcoin ATM." - Bank of Canada
+
+https://user-images.githubusercontent.com/59807305/195987949-aa796718-4f8c-4019-bca6-8d67a9ea3766.jpg
```

Comment:
what is this image url doing here?

Reply:
it's a graph. I saved it and uploaded it.